react-web/src/components/Toast/Toast.tsx

```diff
@@ -4,7 +4,11 @@ const Toast = (props?: any) => {
     return (
         <div className="overlay">
             <div className="toast-wrapper">
+                {props?.title ? (<>
```

Comment:
just curious is there a catch having `<>` with only one child element?

Reply:
Actually, `<>` isn't needed here as it is the only child. Maybe my miss in removing it, as I had multiple elements earlier then combined it into one for now. Though, not aware of any issues in placing `<>` with single child